packages/utils/pack-up/src/__tests__/node.test.ts

```diff
@@ -0,0 +1,343 @@
+import { stripColor } from '../../tests/console';
+import { cleanupWorkspaces, createWorkspace } from '../../tests/workspaces';
+import { init } from '../node/init';
+import { defaultTemplate } from '../node/templates/internal/default';
+import { TemplateOrTemplateResolver, TemplateResolver } from '../node/templates/types';
+
+const loggerMock = {
+  debug: jest.fn(),
+  info: jest.fn(),
+  warn: jest.fn(),
+  // eslint-disable-next-line no-console
+  error: console.error,
```

Comment:
Why not mock this one? We could end up with tests printing errors

Reply:
This is a good question, I think on reflection I was debugging a crash. Let me revert this back to the mock.